content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -0,0 +1,325 @@
+---
+reviewers:
+- tallclair
+title: Pod Security Standards
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Security settings for pods are typically applied by using [security
+contexts](/docs/tasks/configure-pod-container/security-context/). Security Contexts allows for the
+definition of privilege and access controls on a per-pod basis.
```

Comment:
Hello @tallclair .
A few nits:
s/pods/Pods/g
s/pod/Pod/g

Security Contexts allow?

Line 16: Move `A _Pod Security` to the next line?